devito/interfaces.py

```diff
@@ -422,11 +424,14 @@ class DenseData(TensorData):
     """Data object for spatially varying data acting as a :class:`SymbolicData`.
 
     :param name: Name of the symbol
-    :param dtype: Data type of the scalar
-    :param shape: The shape of the tensor
+    :param shape: Domain shape of the associated data for this :class:`Function`.
+                  Note that this does not include the boundary padding added due
+                  the stencil radius for space dimensions.
```

Comment:
Yeah, previously in the other comment I meant something like this

Reply:
Actually, I should remove this one and add it back in once that statement is actually true (not in this PR).